Inc/ReadOxidizerTankPressure.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+extern ADC_HandleTypeDef hadc2;
+
+void readOxidizerTankPressureTask(void const* arg);
+
+uint16_t getAverageAdc2Reading();
```

Comment:
Since it might not always be ADC 2, a better name for this would be `getAverageOxidizerTankPressure()`

But this function adn `uint16_t getAverageAdc2Reading()` are very similar. Maybe we cna make a util function, called `uint16_t averageArray(uint16_t[] array, int size);` and we will have better code reuse. This would go in a new file called `Utils.h` and `Utils.c`.